prog/dftb+/lib_elecsolvers/elsisolver.F90

```diff
@@ -490,7 +490,7 @@ subroutine supportedVersionNumber(major, minor, patch, dateStamp)
       isSupported = .false.
     elseif (major == 2 .and. minor < 6) then
       isSupported = .false.
-    elseif (major == 2 .and. minor == 6 .and. any(patch == [0,1])) then
+    elseif (major == 2 .and. minor == 6 .and. .not.any(patch == [0,1])) then
```

Comment:
`.not.any(patch == [0,1])` === `all(patch /= [0,1])`

Reply:
Yes, it is